src/test/cypress/e2e/support/app-frontend.js

```diff
@@ -89,3 +79,18 @@ Cypress.Commands.add('navigateToTask4', () => {
   cy.get(appFrontend.sendinButton).should('be.visible').click();
   cy.get(appFrontend.confirmContainer).should('be.visible');
 });
+
+Cypress.Commands.add('addItemToGroup', (oldValue, newValue, comment) => {
```

Comment:
Nice reduction of duplication.
The steps are a little bit hard to understand in my opinion.
Especially when chained on the same line.
For debugging purposes, if something breaks, it is in my opinion good practice to have more lines.
Also using temporary variables to describe what state you are in can help with readability.
```js
var addNewItem = cy.get(...);
addNewItem.should(...);
addNewItem.focus();
addNewItem.click();
```
Though it is more lines, it is easier to locate the potential bug.

Reply:
It is not possible in cypress to assign a DOM to a variable using assignment. and it is one of [best practices](https://docs.cypress.io/guides/references/best-practices#Creating-tiny-tests-with-a-single-assertion) of writing cypress tests to chain commands.